### PR TITLE
Basic Documentation and Comments

### PR DESCRIPTION
**Pull request by zachpren0**
-removed bootstrap page

### CODE REVIEW

File: MERN Scripts/mern-emsystem/src/pages/Orderqty.js
Comment (SpeiserK) [inline MERN Scripts/mern-emsystem/src/pages/Orderqty.js:1]: plae

**Review by SpeiserK — APPROVED** [review]
Great work adding these useful comments mate 💯

File: MERN Scripts/mern-emsystem/src/pages/Orderqty.js
Comment (camwlsn) [reply MERN Scripts/mern-emsystem/src/pages/Orderqty.js:1]: plae order form

**Review by camwlsn — APPROVED** [review]
thanks for adding these

**Pull request merged**